images/linux/scripts/installers/apache.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+################################################################################
+##  File:  apache.sh
+##  Desc:  Installs Apache HTTP Server
+################################################################################
+
+# Install Apache
+apt-get install apache2 -y
+
+# Stop apache2.service
+systemctl is-active --quiet apache2.service && systemctl stop apache2.service
```

Comment:
I think we should disable apache and nginx services to start at boot because it can impact on each other and customers builds which use port 80 .

Reply:
@al-cheb is there a reboot in build process? If yes, I will update the PR to disable these.

Reply:
@shivammathur, Thank you. Yep, at the next image deployment the services will be running at boot start.

Reply:
ok. Updated the PR.